programs/whirlpool/Cargo.toml

```diff
@@ -23,6 +23,7 @@ thiserror = "1.0"
 uint = {version = "0.9.1", default-features = false}
 borsh = "0.9.1"
 mpl-token-metadata = { version = "1.7", features = ["no-entrypoint"] }
+solana-security-txt = { version = "^1.1.1" }
```

Comment:
how about use `version = "=1.1.1"`.
I trust Neodyme, but I think we may be a little cautious about running macros in lib.rs.
WDYT ?

Reply:
Yep think pinning is a good idea!